From 92957d84271d8a54604df0a78c4318485c607ef6 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 11 Mar 2023 18:02:04 +0100
Subject: [PATCH 1280/2871] tests: avoid using `test_i18ncmp`

Since `test_i18ncmp` was deprecated in v2.31.*, the instances added in
v2.30.9 needed to be converted to `test_cmp` calls.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 t/t0003-attributes.sh    | 10 +++++-----
 t/t4115-apply-symlink.sh |  2 +-
 t/t6300-for-each-ref.sh  |  6 +++---
 3 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/t/t0003-attributes.sh b/t/t0003-attributes.sh
index 57ba303de8..9d9aa2855d 100755
--- a/t/t0003-attributes.sh
+++ b/t/t0003-attributes.sh
@@ -344,7 +344,7 @@ test_expect_success 'large attributes line ignored in tree' '
 	printf "path %02043d" 1 >.gitattributes &&
 	git check-attr --all path >actual 2>err &&
 	echo "warning: ignoring overly long attributes line 1" >expect &&
-	test_i18ncmp expect err &&
+	test_cmp expect err &&
 	test_must_be_empty actual
 '
 
@@ -357,7 +357,7 @@ test_expect_success 'large attributes line ignores trailing content in tree' '
 	printf "a %02045dtrailing attribute\n" 1 >.gitattributes &&
 	git check-attr --all trailing >actual 2>err &&
 	echo "warning: ignoring overly long attributes line 1" >expect &&
-	test_i18ncmp expect err &&
+	test_cmp expect err &&
 	test_must_be_empty actual
 '
 
@@ -366,7 +366,7 @@ test_expect_success EXPENSIVE 'large attributes file ignored in tree' '
 	dd if=/dev/zero of=.gitattributes bs=101M count=1 2>/dev/null &&
 	git check-attr --all path >/dev/null 2>err &&
 	echo "warning: ignoring overly large gitattributes file ${SQ}.gitattributes${SQ}" >expect &&
-	test_i18ncmp expect err
+	test_cmp expect err
 '
 
 test_expect_success 'large attributes line ignored in index' '
@@ -375,7 +375,7 @@ test_expect_success 'large attributes line ignored in index' '
 	git update-index --add --cacheinfo 100644,$blob,.gitattributes &&
 	git check-attr --cached --all path >actual 2>err &&
 	echo "warning: ignoring overly long attributes line 1" >expect &&
-	test_i18ncmp expect err &&
+	test_cmp expect err &&
 	test_must_be_empty actual
 '
 
@@ -385,7 +385,7 @@ test_expect_success 'large attributes line ignores trailing content in index' '
 	git update-index --add --cacheinfo 100644,$blob,.gitattributes &&
 	git check-attr --cached --all trailing >actual 2>err &&
 	echo "warning: ignoring overly long attributes line 1" >expect &&
-	test_i18ncmp expect err &&
+	test_cmp expect err &&
 	test_must_be_empty actual
 '
 
diff --git a/t/t4115-apply-symlink.sh b/t/t4115-apply-symlink.sh
index 2d03c4e4d1..2b034ff771 100755
--- a/t/t4115-apply-symlink.sh
+++ b/t/t4115-apply-symlink.sh
@@ -72,7 +72,7 @@ test_expect_success SYMLINKS 'symlink escape when creating new files' '
 	cat >expected_stderr <<-EOF &&
 	error: affected file ${SQ}renamed-symlink/create-me${SQ} is beyond a symbolic link
 	EOF
-	test_i18ncmp expected_stderr stderr &&
+	test_cmp expected_stderr stderr &&
 	! test_path_exists .git/create-me
 '
 
diff --git a/t/t6300-for-each-ref.sh b/t/t6300-for-each-ref.sh
index cac7f443d0..7cfa934913 100755
--- a/t/t6300-for-each-ref.sh
+++ b/t/t6300-for-each-ref.sh
@@ -945,9 +945,9 @@ test_failing_trailer_option () {
 	test_expect_success "$title" '
 		# error message cannot be checked under i18n
 		test_must_fail git for-each-ref --format="%($option)" refs/heads/main 2>actual &&
-		test_i18ncmp expect actual &&
+		test_cmp expect actual &&
 		test_must_fail git for-each-ref --format="%(contents:$option)" refs/heads/main 2>actual &&
-		test_i18ncmp expect actual
+		test_cmp expect actual
 	'
 }
 
@@ -966,7 +966,7 @@ test_expect_success 'if arguments, %(contents:trailers) shows error if colon is
 	fatal: unrecognized %(contents) argument: trailersonly
 	EOF
 	test_must_fail git for-each-ref --format="%(contents:trailersonly)" 2>actual &&
-	test_i18ncmp expect actual
+	test_cmp expect actual
 '
 
 test_expect_success 'basic atom: head contents:trailers' '
-- 
2.39.3 (Apple Git-145)

